Applications/Spire/Include/Spire/Ui/SideInputWidget.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef SPIRE_SIDE_INPUT_WIDGET_HPP
+#define SPIRE_SIDE_INPUT_WIDGET_HPP
+#include <QWidget>
+#include "Nexus/Definitions/Side.hpp"
+#include "Spire/Spire/Spire.hpp"
+
+namespace Spire {
+
+  //! Represents a widget for selecting Side values.
+  class SideInputWidget : public QWidget {
```

Comment:
Let's start using descriptive names instead of generic terms. If this is a ComboBox then call it a ```SideComboBox```, if it's a drop down list then ```SideDropDownList```, etc...